plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -1737,6 +1737,7 @@ public void registerCoreMembers() {
         // Specifying a slot will give the player the item to that slot.
         // If an item is already in that slot, the item will not be given
         // unless they are exactly the same items, then it will stack.
+        // Specifying "unlimit_stack_size" will allow an item to stack up to 64.

```

Comment:
A: Should clarify _why_ this might be a relevant object (ie, that some items normally stack to lower limits, like most tools and armor that can only stack to 1.
B: Add the argument to the argument syntax info (line 1727 and line 1764)